src/udl/skel/PHP/pylib/lang_php.py

```diff
@@ -1015,6 +1015,17 @@ def _extra_dirs_from_env(self, env):
             if not pref: continue
             exclude_dirs.update(d.strip() for d in pref.split(os.pathsep)
                                 if exists(d.strip()))
+
+        # Get excluded names
+        excluded_dir_names = [];
+        for excludes_pref in env.get_all_prefs("import_exclude_matches"):
```

Comment:
Can you refactor  uses of `import_exclude_matches` and `phpExcludePaths` to a class variable of `PHPLangIntel` similar to how *lang_python.py* is doing for `PythonLangIntel`?

Reply:
Do you mean to just add a class variable for the strings like this?
```python
importExcludeMatchesPrefName = "import_exclude_matches"
excludePathsPrefName = "phpExcludePaths"
```